tests/libtest/test_rosa.py

```diff
@@ -0,0 +1,13 @@
+import logging
```

Comment:
I don't think we really need to have libtest for simple skipif, anyway you do not testing if it was really skipped or not so this test is not telling anything in the report.

Reply:
I added it to verify it via a valid job. else I can only able to verify this locally. 
I agree this is not any function so I am removing it